### ISSUE TITLE
Bugs in Story Detail Page auto-generated widget

### ISSUE THREAD

**Issue by WendyLBelcher** [PrincetonPEMM/pemm-issues#80]
@ayo-gb Looking good! Just a few little issues.

- [ ] First sentence:
The error in the date sentence is two-fold: 
The only wording appearing is "very recent" even when the story is very old.
Remove "the ... 00s"

Examples of the problem:
This story is very recent: the earliest PEMM manuscript* in which this story appears is from the145000s.
This story is very recent: the earliest PEMM manuscript* in which this story appears is from the151700s.

**Comment by WendyLBelcher** [comment]
@ayo-gb I realized that I had not given you web ready images. The ones I gave you were too large, 1 MB or more, and thus stalling the page. Henok replaced the images across the top (the scribe, manuscript, and Mary) with my new web ones. But I need to give you other images for the Healed Daughter, Thirsty Dog and the King. Well, I think the daughter one is okay, so here are the replacements for the other two.
![Home-page-story-King-image](https://user-images.githubusercontent.com/56000106/197215333-41fecaef-a853-4cb3-925c-e45d4958e292.jpg)
![Home-page-story-Thirsty-Dog-image](https://user-images.githubusercontent.com/56000106/197215334-2e079832-acb1-421b-83a8-1eef8c2b6623.jpg)
